coderd/workspaces.go

```diff
@@ -636,33 +638,58 @@ func createWorkspace(
 		workspaceBuild     *database.WorkspaceBuild
 		provisionerDaemons []database.GetEligibleProvisionerDaemonsByProvisionerJobIDsRow
 	)
+
+	prebuildsClaimer := *api.PrebuildsClaimer.Load()
```

Comment:
nit: I think this could be moved to the `var` declaration inside the tx.

Reply:
done